### DISCUSSION TITLE
System fonts

### DISCUSSION THREAD

**Discussion by laurentpayot** [beercss/beercss#132] [Ideas]
Hi, I discovered https://modernfontstacks.com/ where they maintain _system_ CSS font stacks for different typeface families.

I now use their [*Neo-Grotesque*](https://github.com/system-fonts/modern-font-stacks#neo-grotesque) stack with BeerCSS:  
```css
font-family: Inter, Roboto, 'Helvetica Neue', 'Arial Nova', 'Nimbus Sans', Arial, sans-serif;
```
![Neo-Grotesque](https://user-images.githubusercontent.com/2776142/227151332-f23c38c0-c8b4-43e6-b092-9bd15e669209.png)

On iOS using Helvetica instead of Roboto makes almost no difference:

![Roboto vs Helvetica](https://user-images.githubusercontent.com/2776142/227152106-8f327a47-b693-403c-9ebd-997f56fa5bf5.png)

I’m now saving 20KB of Roboto Flex download (three times the size of my Beercss CSS!) and it is of course GDPR-compliant, it fixes #86 in a new way.
I have never heard of the [Inter font](https://www.figma.com/blog/the-birth-of-inter/) before, but it looks great and is used by GitHub, Mozilla, Figma and many other as a better Roboto.

Material Design specs say you should use your platform fonts when possible anyway, so what do you all think about this?

**Comment by leonardorafael** [comment]
I'm working on it in the next release https://github.com/beercss/beercss/pull/146

**Reply by laurentpayot** [reply]
> I'm working on it in the next release

Awesome! :+1:

**Comment by laurentpayot** [comment]
Thank you @leonardorafael, v3.2.1 is just great!
